cmd/restic/cmd_rewrite.go

```diff
@@ -219,7 +219,7 @@ func runRewrite(ctx context.Context, opts RewriteOptions, gopts GlobalOptions, a
 
 	changedCount := 0
 	for sn := range FindFilteredSnapshots(ctx, snapshotLister, repo, &opts.SnapshotFilter, args) {
-		Verbosef("\nsnapshot %s of %v at %s)\n", sn.ID().Str(), sn.Paths, sn.Time)
+		Verbosef("\nsnapshot %s of %v at %s by %s@%s)\n", sn.ID().Str(), sn.Paths, sn.Time, sn.Username, sn.Hostname)
```

Comment:
From what I can see, we've duplicated that snapshot formatting four times (`cmd_ls` is slightly different). Please just change `internal/restic/snapshot.String()` to output this format and then use that method everywhere.